README.md

```diff
@@ -124,6 +124,7 @@ information on what to include when reporting a bug.
 
 ## Changelog
 
+- Added manual defintion of joystick `SYN_DROPPED` for backwards compatibility with Linux < 2.6.39 (#1196)
```

Comment:
This is a Linux-specific workaround and should be listed under `- [Linux]` (without `Bugfix:`) below.